src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1926,6 +2025,8 @@ static fsal_status_t kvsfs_share_try_new_state(struct kvsfs_fsal_obj_handle *obj
 {
 	fsal_status_t status;
 
+	perfc_trace_inii(PFT_KVSFS_SHARE_TRY_NEW_STATE, PEM_KVSFS_TO_NFS);
```

Comment:
KVSFS tags should not be used from handle.c file.

Reply:
which is convenient function tag to use from handle.c?

Reply:
Pls remove KVSFS sub string

Reply:
fixed